src/Router.js

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
 import Main from './pages/Main/Main';
 import Upload from './pages/Upload/Upload';
+import Login from './pages/Login/Login';
+import LoginKakao from './pages/Login/LoginKakao';
 
 const Router = () => {
   return (
     <BrowserRouter>
       <Routes>
         <Route path="/" element={<Main />} />
         <Route path="/upload" element={<Upload />} />
+        <Route path="/auth/signIn" element={<Login />} />
+        <Route path="/auth/signIn" element={<LoginKakao />} />
```

Comment:
```suggestion
        <Route path="/auth/signIn" element={<LoginKakao />} />
```
Login 컴포넌트에서 LoginKakao를 import 하고 있기 때문에 이 부분의 path 는 불필요합니다.

Reply:
넵 수정했습니다!